Assets/Scripts/QuestEditor/EditorSelectionList.cs

```diff
@@ -133,7 +134,7 @@ public void SelectItem(UnityEngine.Events.UnityAction call)
             if (filtered.Count > i)
             {
                 string key = filtered[i].name;
-                tb = new TextButton(new Vector2(21, offset), new Vector2(20, 1), key, delegate { SelectComponent(key); }, filtered[i].color);
+                tb = new TextButton(new Vector2(21, offset), new Vector2(20, 1), new StringKey(key,false), delegate { SelectComponent(key); }, filtered[i].color);
```

Comment:
There are some things that I'm not sure what are used for. But decided to key all screen texts. For now all text elements need to be stringkey but after quest keys, we can revert some ones.